src/CommonITILTask.php

```diff
@@ -1700,12 +1700,27 @@ public function showForm($ID, array $options = [])
     public function showMassiveActionAddTaskForm()
     {
         $twig = TemplateRenderer::getInstance();
+
         $item = static::getItilObjectItemInstance();
         $item->getEmpty();
+        $this->getEmpty();
+
+        // TODO: it would be better to get options from RichText::getMentionOptions()
+        // here but the only uses of this method are for the massive actions
+        // where we actually have multiples parent objets and thus can't pick a
+        // single one to send to RichText::getMentionOptions().
+        // This mean user mentions won't be available in the massive actions form
+        // until a solution can be found for this.
```

Comment:
If the active profile has `use_mentions == USER_MENTION_RESTRICTED`, we cannot propose the list of selected items actors, it would be too complex. But if the profile has `use_mentions == USER_MENTION_FULL`, we can enable them.

Anyway, having this kind of limitation on massive actions is not an issue.